src/webview/components/options-widget.tsx

```diff
@@ -133,6 +137,30 @@ export class OptionsWidget extends React.Component<OptionsWidgetProps, OptionsWi
         }
     }
 
+    private onContextDropdownChange = (e: DropdownChangeEvent) => {
+        const { setContext, contexts } = this.props;
+        setContext(contexts.filter(context => context.id === Number(e.value))[0]);
+    };
+
+    protected showContexts = () => {
+        if (this.props.contexts.length === 0) {
+            return undefined;
+        }
+        return (
+            <span className='pm-top-label'>
+                <label htmlFor={InputId.Contexts} className='p-inputtext-label'>
+                    Context
+                </label>
+                <Dropdown
+                    id={InputId.Contexts}
+                    value={this.props.context?.id}
+                    options={this.props.contexts}
+                    optionLabel="name"
+                    optionValue="id"
+                    onChange={this.onContextDropdownChange} />
+            </span>);
+    };
```

Comment:
I don't think this needs to be a property declaration:

```suggestion
    protected showContexts() {
        if (this.props.contexts.length === 0) {
            return undefined;
        }
        return (
            <span className='pm-top-label'>
                <label htmlFor={InputId.Contexts} className='p-inputtext-label'>
                    Context
                </label>
                <Dropdown
                    id={InputId.Contexts}
                    value={this.props.context?.id}
                    options={this.props.contexts}
                    optionLabel="name"
                    optionValue="id"
                    onChange={this.onContextDropdownChange} />
            </span>);
    }
```

Handlers are often declared as properties because they need to be passed as callbacks, but this is just called directly.